### ISSUE TITLE
Use TLP Drop-in configs

### ISSUE THREAD

**Issue by KAGEYAM4** [d4nj1/TLPUI#158]
Currently tlpui changes /etc/tlp.conf. Wouldn't it be better if tlpui uses `/etc/tlp.d/*.conf: Drop-in customization snippets, read in lexical (alphabetical) order` ?

As /etc/tlp.conf is part of package in ArchLinux - https://archlinux.org/packages/extra/any/tlp/. Anytime an update drops, users have to merge the conflict.

If tlpui used /etc/tlp.d/<lower-number.conf, this would solve the problem.

**Comment by KAGEYAM4** [comment]
I am little bit conflited, /etc/tlp.conf is very well documented and therefore good place to change config and read about configs.

**Comment by guiodic** [comment]
This should be simple.
1. When first starting tlpui copy /etc/tlp.conf to /etc/tlp.d/99-tlpui.conf and use that as the configuration file.
2. On subsequent starts, if the file 99tlpui.conf already exists, it will not have to perform step 1

**Comment by d4nj1** [comment]
Hi,
Drop-In configurations are currently not handled properly. I would like to enable this but struggle a bit on the "how to visualize". I do not like to confuse users. Ideally you should be able to configure each config item separately wether it should be stored in the default location or in a Drop-In one.

**Comment by KAGEYAM4** [comment]
yeah, i also don't want to complicate this. TPL gets updated ones in a moonlight, so merging config conflicts should not be that big of a deal.
